app/modules/inventory/views/templates/inventory_network_nav.hbs

```diff
@@ -23,5 +28,10 @@
       </div>
     </div>
     <div id="groupsList"></div>
+
+    <div class="groupsInvitationsWrapper hidden">
+      <p class="list-label">{{I18n 'invitations'}}</p>
```

Comment:
What about longer/more explicit text like : "You have been invited to those groups" or "groups invited to join"

Reply:
that would end up taking quite some space on small screens: a longer text could be set in `title` text maybe?

Reply:
sure, i was making this feedback because it did not occur to me straightfully that "invitations" meant "groups youre invited to" and it took me a some time getting the context (although i was fresh and caffeinated)

Reply:
i think those "take some space on small screens" arguments only make sense for a defined language (english) but cannot be applied to all languages anyway